src/Csv.php

```diff
@@ -1310,4 +1311,48 @@ protected function _guess_delimiter($search_depth, $preferred, $enclosure, &$dat
         ksort($filtered);
         $this->delimiter = reset($filtered);
     }
+
+    /**
+     * getCollection
+     * Returns a Illuminate/Collection object
+     * This may prove to be helpful to people who want to
+     * create macros, and or use map functions
+     *
+     * @access public
+     * @link   https://laravel.com/docs/5.6/collections
+     *
+     * @throws ErrorException - If the Illuminate\Support\Collection class is not found
+     * @throws MixedException - If the new Collection class throws an exception
+     * @throws ErrorException - If the collection object is null after initing the collection class
```

Comment:
1. These lines might not be necessary - see below. 
2. Can you link to documentation about the `MixedException`? A quick search for ["mixedexception" "illuminate" "collection"](https://www.google.de/search?q="mixedexception"+"illuminate"+"collection") yielded nothing.
3. While to us humans this looks perfect, to an IDE like NetBeans or PhpStorm it needs a backslash: `\ErrorException`

Reply:
1 - Yes two of the lines are not needed
2 - There is no such thing as mixedexception. I have removed it
3 - I have added the escape character.